server/src/main/java/eu/fasten/server/FastenServer.java

```diff
@@ -299,9 +298,9 @@ private void registerDBConnections(List<DBConnector> dbPlugins) {
     private void loadDependencyGraphResolvers(List<DependencyGraphUser> plugins) {
         plugins.forEach(p -> {
             if (dbUrls == null || depGraphPath == null) {
-                logger.error("Couldn't load dependency graph. Make sure that you have "
-                    + "provided a valid DB URL, username, password, "
-                    + "and a path to the serialized dependency graph.");
+                throw new RuntimeException("Couldn't load dependency graph. Make sure that you have "
```

Comment:
Could you use an IllegalArgumentException instead? RuntimeException is a very general type, the error should communicate the problem.

Reply:
Done!